### ISSUE TITLE
Genes detected in single gene search but not showing in the multigene view

### ISSUE THREAD

**Issue by beamilon** [IGS/gEAR#290]
@adkinsrs
@songeric1107 
I was looking for the following gene list with multigene view in Outer Hair Cell Response to PTS-inducing noise (PrestinCre;RiboTag;CBA mice), RiboTag RNA-seq: slc26a5, ctss, eno1b, c1q1, ppm1h, mmp23.
I was surprised that Slc26a5 did not appear (picture attached)
When looking at the genes individually, Slc26a5 is clearly detected (picture attached)
![image](https://user-images.githubusercontent.com/70719873/164507663-d728507d-8e38-402d-80c6-57d013899305.png)
![image](https://user-images.githubusercontent.com/70719873/164507797-068ef733-9c33-49ce-98c5-d34c4b4b71b8.png)

**Comment by adkinsrs** [comment]
Is this a public dataset in a public profile?  I would like to take a look at this.  That is interesting indeed

**Comment by beamilon** [comment]
Yes it is public.

**Comment by adkinsrs** [comment]
What profile is it in (so I don't have to look through all 30 of the public ones)?

**Comment by beamilon** [comment]
NIHL, cell type-specific transcriptomics, mouse cochlea (Hertzano 2021)

**Comment by adkinsrs** [comment]
A couple of things I have noticed.

- The "c1q1" and "mmp23" genes are not in this dataset (there is a "c1q|1" though). So they should not be plotted.
- If I capitalize the "S" in "Slc26a5" I get that in the heatmap where "mmp23" is currently being shown with the same expression values.  So I think something is up in the multigene curator where gene labels for genes that do not exist in the dataset are replacing valid gene labels.  Interesting issue

**Comment by adkinsrs** [comment]
<img width="614" alt="Screen Shot 2022-04-22 at 9 42 49 AM" src="https://user-images.githubusercontent.com/5665914/164726376-4a40ca0d-f8d6-449b-90a2-f639b1f76171.png">

I can replicate this issue.  I made up a gene "abc" where every other gene in the dataset was valid. In the heatmap "abc" got added to the plot as a label in place of one of the other gene labels. I noticed that if I use the actual labels with capitalization, this did not happen.

If I ensure capitalization is correct with the "Outer Hair Cell Response to PTS-inducing noise" dataset, the proper gene labels show on the heatmap
